src/v3/src/transformer/terminal/odaEnrollment/transformOdaEnrollmentAndroidAppLink.ts

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {
+  ButtonElement,
+  ButtonType,
+  DescriptionElement,
+  FieldElement,
+  IdxStepTransformer,
+  ListElement,
+  StepperButtonElement,
+  TextWithHtmlElement,
+  TitleElement,
+  UISchemaLayout,
+  UISchemaLayoutType,
+} from '../../../types';
+import { copyToClipboard, loc } from '../../../util';
+
+export const transformOdaEnrollmentAndroidAppLink: IdxStepTransformer = ({
+  formBag,
+  transaction,
+}) => {
+  // @ts-expect-error Property 'deviceEnrollment' does not exist on type 'IdxContext' ts(2339)
+  const deviceEnrollment = transaction.context?.deviceEnrollment?.value;
+
+  const { signInUrl } = deviceEnrollment;
+
+  formBag.data.hasOVAccount = 'no';
+
+  formBag.uischema.elements.push({
+    type: UISchemaLayoutType.STEPPER,
+    elements: [
+      {
+        type: UISchemaLayoutType.VERTICAL,
+        elements: [
+          {
+            type: 'Title',
+            options: { content: loc('enroll.title.oda.with.account', 'login') },
+          } as TitleElement,
+          {
+            type: 'Field',
+            key: 'hasOVAccount',
+            translations: [{
+              name: 'label',
+              i18nKey: 'enroll.subtitle.fastpass',
+              value: loc('enroll.subtitle.fastpass', 'login'),
+            }],
+            options: {
+              inputMeta: {
+                name: 'hasOVAccount',
+                options: [
+                  {
+                    value: 'no',
+                    label: loc('enroll.option.noaccount.fastpass', 'login'),
+                  },
+                  {
+                    value: 'yes',
+                    label: loc('enroll.option.account.fastpass', 'login'),
+                  },
+                ],
+              },
+              format: 'radio',
+              name: 'hasOVAccount',
+            },
+          } as FieldElement,
+          {
+            type: 'StepperButton',
+            label: loc('oform.next', 'login'),
+            options: {
+              type: ButtonType.BUTTON,
+              variant: 'primary',
+              nextStepIndex: (widgetContext) => {
+                const { data } = widgetContext;
+
+                if (data.hasOVAccount === 'yes') {
+                  return 1;
+                }
+                return 2;
+              },
+            },
+          } as StepperButtonElement,
+        ],
```

Comment:
```suggestion
        ].map((ele: UISchemaElement) => ({ ...ele, viewIndex: 0 })),
```

Reply:
Which this reminds me, I think we may need to come up with a generic/singular point to add these indices 😢